offboard_control:
    ros__parameters:
      debug: true

      ## Square test
      test: 0
      MotionStart: 15.0
      Tmotion: 15.0
      Trest: 10.0
      StartingPoint: [1.44, 2.75, -0.8]
      deltaX: 1.8
      deltaY: 1.8

      ## Step test
      # test: 1
      # MotionStart: 15.0
      # Tmotion: 10.0
      # Trest: 10.0
      # StartingPoint: [2.41, 4.9, -0.7]      
      # deltaZ: 0.3
